src/components/Comments/Comment.js

```diff
@@ -22,6 +22,14 @@ import BlockComponent from '../BlockComponent';
 import {getCaptionWithLinkStyle} from '../../utils/string/StringUtils';
 import CommentUserName from '../CommentUsername/CommentUsername';
 import ProfilePicture from '../../screens/ProfileScreen/elements/ProfilePicture';
+import SendDMBlack from '../../assets/icons/images/send-dm-black.svg';
+import SendDMAnonBlock from '../../assets/icons/images/send-dm-anon-black.svg';
+import MemoSendDM from '../../assets/icon/SendDM';
+import BottomSheetMenu from '../BottomSheet/BottomSheetMenu';
+import useDMMessage from '../../hooks/core/chat/useDMMessage';
+import useCreateChat from '../../hooks/screen/useCreateChat';
+import {Context} from '../../context';
+import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new imports look fine and seem to be necessary for the new feature of sending direct messages. However, it would be better if you could group them based on their types or sources (e.g., local imports vs library imports) to improve readability.
```diff
 import {getCaptionWithLinkStyle} from '../../utils/string/StringUtils';
 import CommentUserName from '../CommentUsername/CommentUsername';
 import ProfilePicture from '../../screens/ProfileScreen/elements/ProfilePicture';
+import SendDMBlack from '../../assets/icons/images/send-dm-black.svg';
+import SendDMAnonBlock from '../../assets/icons/images/send-dm-anon-black.svg';
+import MemoSendDM from '../../assets/icon/SendDM';
+import BottomSheetMenu from '../BottomSheet/BottomSheetMenu';
+import useDMMessage from '../../hooks/core/chat/useDMMessage';
+import useCreateChat from '../../hooks/screen/useCreateChat';
+import {Context} from '../../context';
+import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->